index.html

```diff
@@ -1,160 +1,278 @@
-            <li><a href="#" class="header_icon"><img src="./image/m_icon.png" alt="m_icon"></a></li>
-            <li><a href="#" class="header_icon"><img src="./image/github_icon.png" alt="github"></a></li>
-            <li><a href="#" class="header_icon"><img src="./image/V_icon.png" alt="V_icon"></a></li>
+          <li>
+            <a href="#" class="header_icon"
+              ><img src="./image/twitter_icon.png" alt="twitter_icon"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="header_icon"
+              ><img src="./image/Linkedin icon.png" alt="linkedin_icon"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="header_icon"
+              ><img src="./image/m_icon.png" alt="m_icon"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="header_icon"
+              ><img src="./image/github_icon.png" alt="github"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="header_icon"
+              ><img src="./image/V_icon.png" alt="V_icon"
+            /></a>
+          </li>
         </ul>
-    </section>  
-    <section class="works">
+      </section>
+      <section class="works">
         <div class="works_card">
-            <img class="works_img" src="./image/works_1.png" alt="works_img">
+          <img class="works_img" src="./image/works_1.png" alt="works_img" />
+          <img class="works_img_desktop" src="./image/desktop/card1.png" alt="works_img_desktop">
+          <div class="sub-works_card">
             <h3 class="works_title">Tonic</h3>
             <ul class="works_options">
-                <li><a href="#" class="works_list1">CANOPY</a></li>
-                <li><img src="./image/Counter.png" alt=""></li>
-                <li><a href="#" class="works_list">Back End Dev</a></li>
-                <li><img src="./image/Counter.png" alt=""></li>
-                <li><a href="#" class="works_list">2015</a></li>           
+              <li><a href="#" class="works_list">CANOPY</a></li>
+              <li><img class="counter" src="./image/Counter.png" alt="" /></li>
+              <li><a href="#" class="works_list">Back End Dev</a></li>
+              <li><img class="counter" src="./image/Counter.png" alt="" /></li>
+              <li><a href="#" class="works_list">2015</a></li>
             </ul>
-            <p class="works_parragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <p class="works_parragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
             <ul class="languages">
-                <li class="lang_skill">html</li>
-                <li class="lang_skill">css</li>
-                <li class="lang_skill">javaScript</li>
```

Comment:
- [ ]  Right now your portfolio should be responsive on all screens but on 800px wide screen the works section is not responsive please make to fix that in order for your portfolio will be responsive

Reply:
Thank you for your observation, but please check that I used two CSS (style.css for the mobile version and desktop.css for the desktop version) and change it when the screen is on 800px